### PR TITLE
Add remote ip logging and some code clean-up

### PR DESCRIPTION
**Pull request by walac**
This PR adds remote ip to requests logging + some code clean-ups.

### CODE REVIEW

File: symLogging.py [hunk 1]
```diff
@@ -47,21 +47,26 @@ def SetLoggingOptions(logOptions):
...
   pid = os.getpid()
-  gLog.log(dbgLevel, "%d\t%s", pid, string)
-
-def LogDebug(string):
+  gLog.log(
+    dbgLevel,
+    "%d\t%s%s",
+    pid,
+    remoteIp + ' ' if remoteIp else '',
+    string)
```
Comment (vdjeric) [inline symLogging.py]: hmm, maybe add it as the last field and use a unique format to make it easier to filter on with grep? so maybe

gLog.log(dbgLevel, "%d\t%s%s", pid, string, " IP=" + remoteIP if remoteIP else "")

File: symbolicationRequest.py [hunk 2]
```diff
@@ -19,7 +19,10 @@ def __init__(self, libName, breakpadId):
     self.libName = libName
     self.breakpadId = breakpadId
 
-def getModuleV2(libName, pdbAge, pdbSig, pdbName):
+def getModuleV2(libName, pdbAge, pdbSig, pdbName, remoteIp):
```
Comment (vdjeric) [inline symbolicationRequest.py]: I'm not concerned about detailed error logging from getModuleV2, just log the IP in the SymbolicationRequest code that calls getModuleV2 if the call fails

File: symbolicationRequest.py [hunk 3]
```diff
@@ -27,37 +30,38 @@ def getModuleV2(libName, pdbAge, pdbSig, pdbName):
...
   pdbAge = (hex(pdbAge)[2:]).lower()
 
   if not isinstance(pdbName, basestring) or not gLibNameRE.match(pdbName):
-    LogDebug("Bad PDB name: " + str(pdbName))
+    Trace("Bad PDB name: " + str(pdbName))
     return None
   return ModuleV3(pdbName, pdbSig + pdbAge)
 
-def getModuleV3(libName, breakpadId):
+def getModuleV3(libName, breakpadId, remoteIp):
```
Comment (vdjeric) [inline symbolicationRequest.py]: Ditto for getModuleV3

**Comment by walac** [comment]
All comments addressed.

**Pull request merged**